app/src/lib/markdown-filters/mention-filter.ts

```diff
@@ -0,0 +1,147 @@
+import { GitHubRepository } from '../../models/github-repository'
+import { getHTMLURL } from '../api'
+import { INodeFilter } from './node-filter'
+
+/**
+ * The Mention Markdown filter looks for user logins and replaces them with
+ * links the users profile. Specifically looking at text nodes and if a
+ * reference like @user is found, it will replace the text node with three nodes
+ * - one being a link.
+ *
+ * Mentions in <pre>, <code> and <a> elements are ignored. Contrary to dotcom
+ * where it confirms whether the user exists or not, we assume they do exist for
+ * the sake of performance (not doing database hits to see if the mentioned user
+ * exists)
+ *
+ * Example: A text node of "That is great @tidy-dev! Good Job!"
+ * Becomes three nodes:
+ * 1) "That is great "
+ * 2) <link src="github.com/tidy-dev">@tidy-dev</link>
+ * 3) "! Good Job!"
+ */
+export class MentionFilter implements INodeFilter {
+  // beginning of string or non-word, non-` char
+  private readonly beginStringNonWord = /(^|[^a-zA-Z0-9_`])/
+
+  // @username and @username_emu for emu support
```

Comment:
What's `emu` here?

Reply:
Enterprise managed user. There was another "emu" usage below (copied from dotcom's notes on their regex) that I swapped for the full usage.